cg/apps/slurm/sbatch.py

```diff
@@ -20,6 +20,34 @@
 log "Running on: $(hostname)"
 """
 
+DRAGEN_SBATCH_HEADER_TEMPLATE = """#! /bin/bash
+#SBATCH --job-name={job_name}
+#SBATCH --partition=dragen
+#SBATCH --account={account}
+#SBATCH --cpus-per-task=24
```

Comment:
Do we still need this many when DRAGEN is doing most of the work?

Reply:
I'm not sure what the right choices are here, I lack the knowledge. I'm following recommendations made by @hassanfa. Would this not be Dragen CPUs?

Reply:
This is correct. All the cores have to be reserved on Dragen

Reply:
Sounds good!